source/time_tables/departure_boards.cpp

```diff
@@ -140,6 +140,90 @@ static void render(PbCreator& pb_creator,
     }
 }
 
+static void render(PbCreator& pb_creator,
```

Comment:
This function is way too loooooooooooooooooooooooooooooong. 
We should definitely break this down into smaller functions. This would ease readability, maintainability and increase modularity for better unit testing.

Reply:
I'm happy to merge this like that if we really need to, but I think we really want to avoid that kind of coding practices ! 

We should really put in practice clean code:
https://alvinalexander.com/programming/clean-code-quotes-robert-c-martin